samza-rest/src/main/java/org/apache/samza/rest/resources/YarnJobResourceConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.rest.resources;
+
+/**
+ * Configurations for the YARN based {@link JobsResource} endpoint.
+ */
+public class YarnJobResourceConfig extends JobsResourceConfig {
```

Comment:
Can we remove this class and have a single variable instead / inline this in JobsResourceConfig

Reply:
JobResourceConfig is agnostic to which cluster manager runs the jobs. Therefore having a YARN related config in JobResourceConfig doesn't seem right.

Reply:
thanks for clarifying.